bin/gbuild

```diff
@@ -329,10 +329,16 @@ build_desc["remotes"].each do |remote|
     if @options[:fetch_branches]
       system!("cd inputs/#{dir} && git fetch -f --update-head-ok #{sanitize_path(remote["url"], remote["url"])} +refs/heads/*:refs/heads/*")
     end
+    commit_fetch = commit
     if @options[:fetch_tags]
       system!("cd inputs/#{dir} && git fetch -f --update-head-ok #{sanitize_path(remote["url"], remote["url"])} +refs/tags/*:refs/tags/*")
+    else
+      refinfo = `cd inputs/#{dir} && git ls-remote #{sanitize_path(remote["url"], remote["url"])} #{commit}`
```

Comment:
another approach is to unconditionally try to retrieve the tag, and ignore failure.  I guess this is a bit cleaner but more complex.